opennms-container/minion/Dockerfile

```diff
@@ -8,11 +8,21 @@ ARG BASE_IMAGE="opennms/deploy-base:jre-1.2.0.b54"
 
 FROM ${BASE_IMAGE} as minion-base
 
+ARG PROM_JMX_EXPORTER_VERSION="0.14.0"
```

Comment:
Looks like 0.15.0 was just released :D

https://mobile.twitter.com/roidelapluie/status/1353748899882741775

Reply:
I just saw it right after creating the PR :) I have it running in my environment and haven't seen any issues. Thanks for the heads up :)